### PR TITLE
Clarify installation steps

### PR DESCRIPTION
**Pull request by mwithi**
Aim is to clarify the installation steps for most common users

### CODE REVIEW

File: doc_admin/AdminManual.adoc [hunk 1]
```diff
@@ -79,26 +79,22 @@ Open Hospital official releases can be downloaded from GitHub: https://github.co
...
+=== 1.6 Unpack
 
-Open Hospital does not require installation. It is sufficient to unzip the downloaded package in a folder of your choice, e.g.:
+NOTE: Do not double-click on the **oh.bat** startup script without unzipping the package first!
 
-* on Linux:
-----
-/home/OH/OpenHospital-[VERSION]
-----
+Most operating systems allow to "open" a downloaded compressed archive by just double-clicking on it.
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc]: Change: `Most operating systems allow to "open"` to `Most operating systems allow one to "open"`

File: doc_admin/AdminManual.adoc [hunk 2]
```diff
@@ -79,26 +79,22 @@ Open Hospital official releases can be downloaded from GitHub: https://github.co
...
-* on Windows:
-----
-C:\Users\OH\OpenHospital-[VERSION]
-----
+In order to work with the application you need to unzip (uncompress) the downloaded package in a folder of your choice.
 
-To uncompress the package:
+To uncompress the package: *Right-click on the package -> Extract all/here*
+
+*Alternative*: to uncompress with command line instructions
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc]: add a colon (:) at the end of the line

File: doc_admin/AdminManual.adoc [hunk 3]
```diff
@@ -135,24 +115,38 @@ NOTE: The information needed to use the OH software is not included in this manu
 
 == 2 Installation
 
+Open Hospital is a client application that can be used in two modalities: 
+
+* <<client,2.1 OH - CLIENT mode>>: multi computers setup, with a central database (requires a little IT knowledge)
+* <<portable,2.2 OH - PORTABLE mode>>: single computer setup, with inbuilt databases (click and run)
+
+TIP: Portable mode can be also used to test the application
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc]: add period (.) at the end of the line

File: doc_admin/AdminManual.adoc [hunk 4]
```diff
@@ -135,24 +115,38 @@ NOTE: The information needed to use the OH software is not included in this manu
 
 == 2 Installation
 
+Open Hospital is a client application that can be used in two modalities:
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc]: `used in two modalities:` might be a little clearer as `used in two different ways:`   This makes it just a little more readable.

File: doc_admin/AdminManual.adoc [hunk 5]
```diff
@@ -135,24 +115,38 @@ NOTE: The information needed to use the OH software is not included in this manu
...
+* <<client,2.1 OH - CLIENT mode>>: multi computers setup, with a central database (requires a little IT knowledge)
+* <<portable,2.2 OH - PORTABLE mode>>: single computer setup, with inbuilt databases (click and run)
+
+TIP: Portable mode can be also used to test the application
+
+[#client]
 === 2.1 OH - CLIENT mode
 
-Open Hospital is a client application that can be used on a single computer (PORTABLE mode) or in a client/server configuration (CLIENT mode). The server part utilizes a DBMSfootnote:[DataBase Management System] to archive, share, and present the information. The free and open-source MariaDBfootnote:[See https://www.mariadb.org] is the preferred database server.
+In this modality, the application will need a Server with a DBMSfootnote:[DataBase Management System] to store the data.
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc]: change `modality` to `mode`

File: doc_admin/AdminManual.adoc [hunk 6]
```diff
@@ -135,24 +115,38 @@ NOTE: The information needed to use the OH software is not included in this manu
...
+* Server 
+** <<download,Download>> the Open Hospital package for the selected Operating System (see <<prereq,Prerequisites>>)
+** <<unpack,Unpack>> the downloaded package on the server machine by uncompressing the archive in a folder of your choice
+** <<database-server,Install and configure MariaDB/MySQL>> on the server/computer that will act as the database server
+** <<ohdatabase-creation,Create the database>> through the execution of a script found in the _sql/_ folder
+** <<setup,Test>> and <<run,run>> the application using 'localhost' setting
+* Clients
+** Unpack the downloaded package on a client machine in a folder of your choice
+** <<setup,Setup>> and <<run,run>> the application to point to the database server using the hostname or the IP address
+** Copy the so configured package on every client
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc]: I would change `so` to `newly`

File: doc_admin/AdminManual.adoc [hunk 7]
```diff
@@ -407,11 +403,11 @@ Specify, according to the values specified in the installation phase:
 
 NOTE: These settings are required for every Open Hospital installation/client.
 
-More information on these files can be found in the sections <<database-properties,database.properties>> and
-<<log4j-properties,log4j.properties>>.
+More information on these files can be found in the sections under <<configuration,3. Configuration>> section.
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc]: Isn't the trailing `section` after the reference redundant? 
What about: 
`More information on these files can be found in the <<configuration,3. Configuration>> section.`

File: doc_admin/AdminManual.adoc (hunk 1 above)
Comment (mwithi) [reply doc_admin/AdminManual.adoc]: ok, thanks!

File: doc_admin/AdminManual.adoc (hunk 2 above)
Comment (mwithi) [reply doc_admin/AdminManual.adoc]: two subsquent colons? line 94 is meant to be more like a title (but not indented like other titles)

File: doc_admin/AdminManual.adoc (hunk 3 above)
Comment (mwithi) [reply doc_admin/AdminManual.adoc]: alright, thanks!

File: doc_admin/AdminManual.adoc (hunk 4 above)
Comment (mwithi) [reply doc_admin/AdminManual.adoc]: good, thanks!

File: doc_admin/AdminManual.adoc (hunk 5 above)
Comment (mwithi) [reply doc_admin/AdminManual.adoc]: ok, thanks!

File: doc_admin/AdminManual.adoc (hunk 6 above)
Comment (mwithi) [reply doc_admin/AdminManual.adoc]: I agree! thanks!

File: doc_admin/AdminManual.adoc (hunk 7 above)
Comment (mwithi) [reply doc_admin/AdminManual.adoc]: absolutely, thanks!

File: doc_admin/AdminManual.adoc (hunk 1 above)
Comment (mizzioisf) [reply doc_admin/AdminManual.adoc]: I will definitely leave "setup" and not "unpack".
I wouldn't mention a general note about the most operating system, I would just leave the instructions plain and simple. We have already removed a lot of "conversational" informations from the admin manual.

File: doc_admin/AdminManual.adoc (hunk 2 above)
Comment (mizzioisf) [reply doc_admin/AdminManual.adoc]: Ok, I would write "Alternative method: uncompress package from command line"

File: doc_admin/AdminManual.adoc (hunk 6 above)
Comment (mizzioisf) [reply doc_admin/AdminManual.adoc]: Ok

File: doc_admin/AdminManual.adoc [hunk 8]
```diff
@@ -153,6 +153,16 @@ Before starting, check the following software compatibility matrix, to verify th
...
+
+Depending on the operating system, there are different ways to install the Java Runtime Environment (JRE).
+The **oh.sh / oh.bat** scripts can be used to download and install the latest JRE available; it is also possible to install it
+manually, following the specific instructions for the operating system being used.
+
+NOTE: See Software Compatibility Matrix to identify the correct Java version.
+
+
 [#database-server]
 ==== 2.1.3 Database Server
```
Comment (mizzioisf) [inline doc_admin/AdminManual.adoc]: Was this paragraph missing/removed ?

File: doc_admin/AdminManual.adoc [hunk 9]
```diff
@@ -84,7 +84,7 @@ More information can be found on the Open Hospital website: https://www.open-hos
 
 NOTE: Do not double-click on the **oh.bat** startup script without unzipping the package first!
 
-Most operating systems allow to "open" a downloaded compressed archive by just double-clicking on it. 
+Most operating systems allow one to "open" a downloaded compressed archive by just double-clicking on it.
```
Comment (mizzioisf) [inline doc_admin/AdminManual.adoc]: Together with @dbmalkovsky we have spent quite some time removing all the "one" reference from the manual. Are we reverting that ? :-(

File: doc_admin/AdminManual.adoc [hunk 10]
```diff
@@ -403,7 +403,7 @@ Specify, according to the values specified in the installation phase:
 
 NOTE: These settings are required for every Open Hospital installation/client.
 
-More information on these files can be found in the sections under <<configuration,3. Configuration>> section.
+More information on these files can be found in the <<configuration,3. Configuration>> section.
```
Comment (mizzioisf) [inline doc_admin/AdminManual.adoc:405]: good !

File: doc_admin/AdminManual.adoc [hunk 11]
```diff
@@ -144,7 +144,7 @@ The Administrator/Installer tasks are hereby listed:
 * Clients
 ** Unpack the downloaded package on a client machine in a folder of your choice
 ** <<setup,Setup>> and <<run,run>> the application to point to the database server using the hostname or the IP address
-** Copy the so configured package on every client
+** Copy the newly configured package on every client
```
Comment (mizzioisf) [inline doc_admin/AdminManual.adoc:147]: agreed and also agree on all the mode/modality change

**Review by mizzioisf — COMMENTED** [review]
See my inline comments

File: doc_admin/AdminManual.adoc (hunk 9 above)
Comment (dbmalkovsky) [reply doc_admin/AdminManual.adoc]: Good point!   How about: `A downloaded compressed archive can be opened on most operating systems by just double-clicking on it.`     That gets rid of the `one` reference.

File: doc_admin/AdminManual.adoc (hunk 1 above)
Comment (mwithi) [reply doc_admin/AdminManual.adoc]: Maybe now it's too much "formal" that few understand it, apparently... :)

IMHO, if users don't understand, the instructions are useless, no matter how "savy" and "tech" they are.

We have seen that many users double-click from inside the archive instead to unpack it, so I believe that information is very useful and relevant for our target users, also not at all "conversational". 

The instrutions to unpack a zip file from command line in Windows, it is "plain and simple" like to "screw in a bolt with a hammer" :)

Also, why to call "setup" something that is actually just unpacking? Setup what?

File: doc_admin/AdminManual.adoc (hunk 8 above)
Comment (mwithi) [reply doc_admin/AdminManual.adoc]: which one?

File: doc_admin/AdminManual.adoc (hunk 9 above)
Comment (mwithi) [reply doc_admin/AdminManual.adoc]: ok for me

File: doc_admin/AdminManual.adoc (hunk 1 above)
Comment (mwithi) [reply doc_admin/AdminManual.adoc]: > See my inline comments

I don't know why some comments are messed up, referring to different commits maybe.
Let's start over from the [Files Changed](https://github.com/informatici/openhospital-doc/pull/131/files) page.

Some comments changes applied though.

**Comment by mwithi** [comment]
> See my inline comments

I don't know why some comments are messed up, referring to different commits maybe.
Let's start over from the [Files Changed](https://github.com/informatici/openhospital-doc/pull/131/files) page.

Some comments changes applied though, two remaining.

File: doc_admin/AdminManual.adoc (hunk 8 above)
Comment (mwithi) [reply doc_admin/AdminManual.adoc]: I think it's a no longer existing change.

**Review by mizzioisf — APPROVED** [review]

**Comment by mwithi** [comment]
As "agreed" via other channels, we merge this as is. Further improvements are still possible.

**Pull request merged**